crates/matrix-sdk/src/room_preview.rs

```diff
@@ -119,7 +123,7 @@ impl RoomPreview {
 
     /// Create a room preview from a known room (i.e. one we've been invited to,
     /// we've joined or we've left).
-    pub(crate) async fn from_known(room: &Room) -> Self {
+    pub async fn from_known(room: &Room) -> Self {
```

Comment:
Why is this public for everyone now? This is an internal method called from `get_room_preview`, and I think it should remain as such.

Reply:
Reverted, sorry. I think I changed it for some tests outside the crate and forgot to put it back the way it was before.